android/build.gradle

```diff
@@ -8,19 +8,43 @@ android {
     compileSdkVersion safeExtGet('compileSdkVersion', 28)
     buildToolsVersion safeExtGet('buildToolsVersion', '28.0.3')
 
+    compileOptions {
+        targetCompatibility JavaVersion.VERSION_1_8
```

Comment:
Is this required? We might run into issues on older Android devices that only support Java 7 features, since the issue will no longer be caught by the compiler. This change itself shouldn't cause issues, but if a developer uses Java 8 features then it might not work on Android 7 and older devices.

Reply:
I am not sure, but I think yes ... This is not my code, it already there in master branch :/ I just did a " git mv "

Reply:
It was introduced in version : 5.1.0-alpha7